readme.md

```diff
@@ -24,6 +25,7 @@
 
 
 ## Mini Projects
+- [BackdropLayout](https://github.com/roiacult/BackdropLayout) - Android Library for creating awesome Backdrop View only with xml
```

Comment:
thanks for your contribution @roiacult 
I think this project should be listed in Extracurricular Projects category, since it was not an course assignment (mini-project) neither a multidisciplinary projects
move it to the right category (new commit, or you can amend your last commit), and let me now when you finish in order to merge the your PR

Reply:
thanks for clarifying @philomath213 
i moved the library to Extracurricular Projects category as you suggested